cmd/security-agent/app/app.go

```diff
@@ -89,7 +91,8 @@ Datadog Security Agent takes care of running compliance and security checks.`,
 )
 
 func init() {
-	SecurityAgentCmd.PersistentFlags().StringArrayVarP(&confPathArray, "cfgpath", "c", nil, "path to a yaml configuration file")
+	var defaultConfPathArray = []string{path.Join(commonagent.DefaultConfPath, "datadog.yaml")}
```

Comment:
the default is `datadog.yaml` instead of `security-agent.yaml` for backward compatibility?

Reply:
Exactly.

For this release, configuration parameters could be in both files. In later releases, we want to move the parameters used by the `security-agent` in its own configuration file and only loads them for this file.